docs/website/blog/2023-01-04-odo-v3.5.0.md

```diff
@@ -0,0 +1,119 @@
+---
+title: odo v3.5.0 Released
+author: Philippe Martin
+author_url: https://github.com/feloy
+author_image_url: https://github.com/feloy.png
+tags: ["release"]
+slug: odo-v3.5.0
+---
+
+odo `v3.5.0` is now out!
+
+<!--truncate-->
+
+To install `odo`, follow [the installation guide](../docs/overview/installation).
+
+## Notable Changes
+Check this Playlist for an overview of the most notable changes in this release: [odo v3.5.0 (Sprint 229)](https://www.youtube.com/playlist?list=PLGMB2PY4SNOoGCUFCbqxnQclhnHA0L73D)
+
+### Features
+
+#### `odo delete component` deletes component running on podman
+
+When you run `odo dev` on podman platform, pressing ctrl-c will delete the resources from podman. 
+
+If `odo dev` does not delete the reosurces as expected (because it failed unexpectedly, it has been killed with the KILL(9) signal, etc), you can use `odo delete component` to delete the resources. 
```

Comment:
```suggestion
When you run `odo dev` on podman platform, pressing Ctrl+c will delete the resources from podman. If `odo dev` does not delete the resources as expected (because it failed unexpectedly, it has been killed with the KILL(9) signal, etc), you can use `odo delete component` to delete the resources. 
```

I find the extra line less readable. I would prefer we remove it.